packages/opencensus-core/src/trace/model/tracer-cls.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as cls from '../../internal/cls';
+
+import {NoRecordSpan} from './no-record/no-record-span';
+import {CoreTracer} from './tracer';
+import * as types from './types';
+
+/**
+ * This class represent a tracer with CLS.
```

Comment:
Explain what CLS is and when you would or would not want to use it.

Reply:
+1